### PR TITLE
Dsr209 introduction to GitHub

### PR DESCRIPTION
**Pull request by DrakeRobertson**
I updated my codingame code to reflect the changes suggested by Tate.

### CODE REVIEW

File: thorgame/dsr209/thor_codingame.cpp [hunk 1]
```diff
@@ -2,15 +2,13 @@
 //
```
Comment (bjones1) [inline thorgame/dsr209/thor_codingame.cpp:2]: Start with the title of the game.

File: thorgame/dsr209/thor_codingame.cpp [hunk 2]
```diff
@@ -120,18 +105,16 @@ int main()
         // it fails that but it is already in this if statement, then Thor's
         // direction can be determined to be north without the need to do any
         // more comparisons.
-        
-        if(light_x == initial_tx){
-            if(light_y > initial_ty){
+        if (light_x == thor_x) {
```
Comment (bjones1) [inline thorgame/dsr209/thor_codingame.cpp:108]: The paragraph above seems unfinished.

**Review by bjones1 — CHANGES_REQUESTED** [review]

File: thorgame/dsr209/thor_codingame.cpp (hunk 1 above)
Comment (DrakeRobertson) [reply thorgame/dsr209/thor_codingame.cpp:7]: I have updated this to include the title of the game at the beginning.

File: thorgame/dsr209/thor_codingame.cpp (hunk 2 above)
Comment (DrakeRobertson) [reply thorgame/dsr209/thor_codingame.cpp:114]: I added to the end of that paragraph to close it out a little better.

**Pull request merged**